docs/changes.rst

```diff
@@ -5,19 +5,22 @@ Changelog
 =====
 
 * Added concept for closable stores.
+
   * Stores and Decorators can now be opened using with KeyValueStore as store
   * Implemented this functionality for baseclasses and the AzureBlockBlobStore
 
 1.4.4
 =====
 
-* Keys are not quoted for ``FSSpecStore``s anymore.
+* Keys are not quoted for ``FSSpecStore``\s anymore.
```

Comment:
Why the backslash here?

Reply:
See [here](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#character-level-inline-markup-1)